### DISCUSSION TITLE
My commits don't count as contributions

### DISCUSSION THREAD

**Discussion by JIMMUGUNA789** [community/community#44989] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My commits are not being counted as contributions on Git Hub. I have two email addresses: a primary email address and a student email address. My commit email address is set to the primary email address, and the email address is set to public. How can I get my commits to be counted as contributions?

**Comment by airtower-luna** [comment, marked as answer]
It's impossible to answer without seeing the repository in question, but the documentation has a list of the exact conditions a commit has to meet to count: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile
